### PR TITLE
Make untar extract message only when DEBUG=true

### PR DESCRIPTION
**Pull request by purbon**
small clean up improvement.

### CODE REVIEW

**Comment by jsvd** [comment]
since this a rake task, maybe we could check if rake was called with -v (verbose)?

**Comment by purbon** [comment]
@jsvd all this options are lost because of the restart with the internal rake, so we can not do that.

File: rakelib/z_rubycheck.rake [hunk 1]
```diff
@@ -1,6 +1,6 @@
 if ENV['USE_RUBY'] != '1'
   if RUBY_ENGINE != "jruby" or Gem.ruby !~ /vendor\/jruby\/bin\/jruby/
-    puts "Restarting myself under Vendored JRuby (currently #{RUBY_ENGINE} #{RUBY_VERSION})"  if $DEBUG
+    puts "Restarting myself under Vendored JRuby (currently #{RUBY_ENGINE} #{RUBY_VERSION})"  if ENV['DEBUG']
```
Comment (jsvd) [inline rakelib/z_rubycheck.rake]: take this opportunity to remove the extra space before the "if" :)

**Comment by jsvd** [comment]
LGTM

**Pull request closed without merging**